### PR TITLE
Update for Carthage distribution

### PR DESCRIPTION
**Pull request by xNekOIx**
Please accept and add new tag (or move 0.1 to latest commit). :cactus:

### CODE REVIEW

File: Sodium/Sign.swift [hunk 1]
```diff
@@ -38,7 +38,7 @@ public class Sign {
         return KeyPair(publicKey: PublicKey(data: pk!), secretKey: SecretKey(data: sk!))
     }
     
-    public func keyPair(seed seed: NSData) -> KeyPair? {
+    public func keyPair(#seed: NSData) -> KeyPair? {
```
Comment (jedisct1) [inline Sodium/Sign.swift]: This syntax doesn't exist any more since Swift 2.

Comment (xNekOIx) [inline Sodium/Sign.swift]: xcode 7 has not yet released. 
what is correct function signature will be in this case?
```public func keyPairWithSeed(seed: NSData) -> KeyPair?``` ?

Comment (jedisct1) [inline Sodium/Sign.swift]: `seed seed` works fine under Xcode 7, so I'd rather keep that since it also works on Xcode 6 minus a warning.

**Comment by xNekOIx** [comment]
bump :cake:

**Comment by jedisct1** [comment]
Done :)

**Comment by xNekOIx** [comment]
:+1:

**Pull request merged**